### PR TITLE
Host header fallback

### PR DESCRIPTION
**Pull request by kolchurinvv**
after a recent update host_header is not defined / empty string ... for some reason

### CODE REVIEW

**Comment by kolchurinvv** [comment]
solves #39

File: src/handler.js [hunk 1]
```diff
@@ -148,6 +148,7 @@ function ssr(req) {
  */
 function get_origin(headers) {
   const protocol = (protocol_header && headers.get(protocol_header)) || "https";
+  host_header = host_header || "host";
```
Comment (silvio2402) [inline src/handler.js:151]: I notice you're providing a default value for `host_header`. Consider utilizing the `env` function's fallback parameter on line 23 to set "Host" as the fallback. For example: `const host_header = env("HOST_HEADER", "Host").toLowerCase();`
This approach keeps default configurations centralized and also aligns with the common practice of capitalizing standard HTTP header names.

**Review by silvio2402 — COMMENTED** [review]
I notice you're providing a default value for host_header. Consider utilizing the env function's fallback parameter on line 23 to set "Host" as the fallback.
For example: const host_header = env("HOST_HEADER", "Host").toLowerCase();
This approach keeps default configurations centralized.
Thanks for your efforts on this!

**Comment by kolchurinvv** [comment]
@silvio2402 thank you for the advice. makes sense. applied the change

**Comment by catdadcode** [comment]
I've forked this repo and published my own NPM package. The fix for this issue is already merged into my fork and I plan to keep it updated. Anyone else having this issue is free to use it if you'd like:

```zsh
bun add @catdadcode/svelte-adapter-bun -D
```

https://github.com/catdadcode/svelte-adapter-bun

I've also added signal event handling to the fork. Previously if you ran your Svelte app in a Docker container then killing the container would take 10+ seconds before Docker would force kill it. The resulting `build/` app from this adapter now has proper process signal handling and will gracefully shutdown immediately upon receiving any of `SIGTERM`, `SIGINT`, `SIGQUIT`, `SIGSTOP`, or `SIGKILL`.

**Comment by silvio2402** [comment]
> I've forked this repo and published my own NPM package. The fix for this issue is already merged into my fork and I plan to keep it updated. Anyone else having this issue is free to use it if you'd like:

Thanks for your efforts. It'd also be great if you could make a PR in this repository so all users can benefit from your changes.

**Comment by catdadcode** [comment]
> > I've forked this repo and published my own NPM package. The fix for this issue is already merged into my fork and I plan to keep it updated. Anyone else having this issue is free to use it if you'd like:
> 
> Thanks for your efforts. It'd also be great if you could make a PR in this repository so all users can benefit from your changes.

I'll be happy to do so if this repo starts rolling in changes again. As it stands [Bun is linking over to my fork for now](https://github.com/oven-sh/awesome-bun/commit/aa281af5ab09246df1f8b6628d282b5e9a79c91b). I'll be rewriting this adapter a bit as well, including adding tests.

**Comment by dimdenGD** [comment]
@gornostay25 Why you're not merging this? Currently every single person who installs your adapter gets this error and has to look into open issues and then find this PR and get the fork instead.

**Comment by kolchurinvv** [comment]
I'd just switch to the adapter maintained by @catdadcode  😃

**Comment by dimdenGD** [comment]
It'd probably be nice to include that in the Readme

**Comment by gornostay25** [comment]
Fixed: https://github.com/gornostay25/svelte-adapter-bun/pull/47

**Pull request closed without merging**